2.js

```diff
@@ -1,3 +1,7 @@
 console.log("esta es la archivo 2")
 console.log("esta es un funcionalidad b")
 console.log("esta es la archivo 3")
+
+
+
+este es un mensaje
```

Comment:
```suggestion
console.log("este es un mensaje")
```
Use el console.log ome